src/ui/components/Input/story/input.js

```diff
@@ -52,12 +52,17 @@ function storyInput(selectedStory) {
 			icon: [
 				{
 					attr: {
-						"data-feather": "eye",
+						class: "fa-regular fa-envelope",
 					},
 					size: "small",
-					position: "right",
+					position: "left",
 				},
 			],
+			requiredText: "Ingresa el dato por favor",
```

Comment:
En la configuracion por defecto, esta propiedad tiene que tener el valor en ingles `requiredText: "This is a required field."`